Makefile

```diff
@@ -5,7 +5,7 @@ ALL:
 	npm i
 
 check:
-	npm t
+	-find test/*.js test/standalone/*.js | xargs -n 1 node | node_modules/tap-spec/bin/cmd.js
```

Comment:
to avoid linux edge case using `make`. hyphen suppresses an improper exit code, nice catch